solution/ui/regulations/css/scss/partials/_header.scss

```diff
@@ -358,5 +361,130 @@ header {
                 }
             }
         }
+
+        .header--sign-in {
+            height: $header_input_height_wide;
+            margin-left: 1rem;
+
+            &:has(> a) {
+                margin-top: auto;
```

Comment:
we can now use the very handy `:has()` selector: https://frontendmasters.com/blog/what-you-need-to-know-about-modern-css-spring-2024-edition/#the-has-pseudo-selector